.envrc

```diff
@@ -0,0 +1,12 @@
+export SECURITY_SALT=development
+export PATH="$PWD/bin/:$PATH"
+
+if [ ! -d vendor ]; then
```

Comment:
I think we can remove the if check. If the composer.lock file was updated, you might want to install the newest dep versions by running `composer install` again.